cloud/linode/loadbalancers.go

```diff
@@ -233,6 +267,10 @@ func (l *loadbalancers) deleteUnusedConfigs(ctx context.Context, nbConfigs []lin
 //
 // EnsureLoadBalancerDeleted will not modify service.
 func (l *loadbalancers) EnsureLoadBalancerDeleted(ctx context.Context, clusterName string, service *v1.Service) error {
+	ctx = sentry.SetHubOnContext(ctx)
+	sentry.SetTag(ctx, "cluster_name", clusterName)
+	sentry.SetTag(ctx, "service", service.Name)
+
 	_, exists, err := l.GetLoadBalancer(ctx, clusterName, service)
 	if err != nil {
 		return err
```

Comment:
this error is worth capturing as well

Reply:
`GetLoadBalancer` already captures errors, since it can also be called on its own, which is why the error isn't captured there. I should document that, though.

Reply:
right, I see that, this looks good then.